src/cascadia/TerminalSettingsModel/MTSMSettings.h

```diff
@@ -138,8 +138,12 @@ Author(s):
 #define MTSM_THEME_SETTINGS(X)                                                             \
     X(winrt::Microsoft::Terminal::Settings::Model::WindowTheme, Window, "window", nullptr) \
     X(winrt::Microsoft::Terminal::Settings::Model::TabRowTheme, TabRow, "tabRow", nullptr) \
+    X(winrt::Microsoft::Terminal::Settings::Model::SettingsTheme, Settings, "settings", nullptr) \
     X(winrt::Microsoft::Terminal::Settings::Model::TabTheme, Tab, "tab", nullptr)
 
+#define MTSM_THEME_SETTINGS_SETTINGS(X)                                                                                              \
+    X(winrt::Windows::UI::Xaml::ElementTheme, RequestedTheme, "applicationTheme", winrt::Windows::UI::Xaml::ElementTheme::Default)

```

Comment:
nit: we may want to give this a different name. I know it's exactly the same as `applicationTheme`, but like, `uiTheme`? Something... smaller, as to indicate it's not controlling the whole app, only the page?

(we'll do wordsmithing in team discussion)